src/lsst/cmservice/web_app/app.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+from typing import Annotated
+import traceback
+from collections.abc import AsyncGenerator
+from contextlib import asynccontextmanager
+
+from fastapi import APIRouter, FastAPI, Request, Depends, Form
+from fastapi.responses import HTMLResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+
+from lsst.cmservice import db
+from safir.dependencies.arq import arq_dependency
+from safir.dependencies.db_session import db_session_dependency
+from safir.dependencies.http_client import http_client_dependency
+from sqlalchemy.ext.asyncio import async_scoped_session
+from safir.logging import configure_logging, configure_uvicorn_logging
+
+from lsst.cmservice.config import config
+from lsst.cmservice.web_app.pages.campaigns import search_campaigns, get_campaign_details
+from lsst.cmservice.web_app.pages.steps import get_campaign_steps, get_step_details
+from lsst.cmservice.web_app.pages.step_details import get_step_details_by_id
+
+
+configure_logging(profile=config.profile, log_level=config.log_level, name=config.logger_name)
```

Comment:
Are these global inits needed/desireable here, since already down in `main.py`?

Reply:
I'll double check they are needed here and fix it if needed in the next PR